app/controllers/conferences_controller.rb

```diff
@@ -9,21 +9,21 @@ def redirect
   end
 
   private
+ 
+  def conferences
+    @conferences ||= Conference.ordered(sort_params).in_current_season
+  end
+  helper_method :conferences
```

Comment:
Personal preference, but since this will probably not be used outside the `index` action, I'd really like to see the `@conferences` being explicitly defined there, rather then using the `helper_method` macro in the view. I generally don't like this pattern and this something like this is way more obvious, especially for newcomers to the app:

```ruby
def index
  @conferences = confereces
end

private

def conferences
  Conference.ordered etc
end

# in the view:
= render 'conferences/table', namespace: :orga, conferences: @conferences
```